### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by rladenson**
Please ignore the first two commits where I messed up the second one reverts the first.

### CODE REVIEW

File: docs/content/command-list.md [hunk 1]
```diff
@@ -96,11 +96,12 @@ Words in **\<angle brackets>** or **[square brackets]** mean fill-in-the-blank.
 
 ## Utility
 - `pk;random [-group]` - Shows the info card of a randomly selected member [or group] in your system.
-- `pk;message <message id / message link>` - Looks up information about a proxied message by its message ID or link.
+- `pk;message <message id / message link / reply>` - Looks up information about a proxied message by its message ID or link.
 - `pk;invite` - Sends the bot invite link for PluralKit.
 - `pk;import` - Imports a data file from PluralKit or Tupperbox.
 - `pk;export` - Exports a data file containing your system information.
 - `pk;permcheck [server id]` - [Checks the given server's permission setup](./staff/permissions.md#permission-checker-command) to check if it's compatible with PluralKit.
+- `pk;edit [message link / proxy]` - Edits a proxied message. Default edits the last proxied message in that channel. NOTE: THIS DOES NOT SUPPORT MESSAGE IDS.
```
Comment (spiralw) [inline docs/content/command-list.md:104]:
```suggestion
- `pk;edit [message link / reply]` - Edits a proxied message. Default edits the last proxied message in that channel. NOTE: THIS DOES NOT SUPPORT MESSAGE IDS. 
```

File: docs/content/user-guide.md [hunk 2]
```diff
@@ -612,3 +612,7 @@ pk;import https://link/to/the/data/file.json
 
 ### Exporting your PluralKit data
 To export all the data associated with your system, run the `pk;export` command. This will send you a JSON file containing your system, member, and switch information.
+
+## Reply commands
+Commands that work with message links or ids will sometimes also work by replying to a message. If you use this method you do not need to use the message link/id field. You can tell if a command works with replies if it has <reply> as a field in the command in the [Command List](../commands).
```
Comment (spiralw) [inline docs/content/user-guide.md:617]: Are there any commands that require a message argument that do not work with a reply?

**Review by spiralw — CHANGES_REQUESTED** [review]

File: docs/content/user-guide.md (hunk 2 above)
Comment (rladenson) [reply docs/content/user-guide.md:617]: Currently no. And I guess if we made any that didn't we could change it.

**Comment by rladenson** [comment]
Closed bc I want to make some more changes since I was not the most coherent while doing this and forgot a couple other things I wanted to do.

**Comment by spiralw** [comment]
Merged in 1fa2d9a093c804006e2a2928817e24c7750b3aa1

**Pull request closed without merging**